src/app/util/af-enums.h

```diff
@@ -24,44 +24,49 @@
-    EMBER_ZCL_STATUS_ACTION_DENIED               = 0x93,
-    EMBER_ZCL_STATUS_TIMEOUT                     = 0x94,
-    EMBER_ZCL_STATUS_ABORT                       = 0x95,
-    EMBER_ZCL_STATUS_INVALID_IMAGE               = 0x96,
-    EMBER_ZCL_STATUS_WAIT_FOR_DATA               = 0x97,
-    EMBER_ZCL_STATUS_NO_IMAGE_AVAILABLE          = 0x98,
-    EMBER_ZCL_STATUS_REQUIRE_MORE_IMAGE          = 0x99,
-    EMBER_ZCL_STATUS_NOTIFICATION_PENDING        = 0x9A,
-    EMBER_ZCL_STATUS_HARDWARE_FAILURE            = 0xC0,
-    EMBER_ZCL_STATUS_SOFTWARE_FAILURE            = 0xC1,
-    EMBER_ZCL_STATUS_UNSUPPORTED_CLUSTER         = 0xC3,
-    EMBER_ZCL_STATUS_LIMIT_REACHED               = 0xC4,
-    EMBER_ZCL_STATUS_NEEDS_TIMED_INTERACTION     = 0xC6,
+    EMBER_ZCL_STATUS_SUCCESS                  = 0x00,
+    EMBER_ZCL_STATUS_FAILURE                  = 0x01,
+    EMBER_ZCL_STATUS_INVALID_SUBSCRIPTION     = 0x7D,
+    EMBER_ZCL_STATUS_UNSUPPORTED_ACCESS       = 0x7E,
+    EMBER_ZCL_STATUS_NOT_AUTHORIZED           = 0x7E, // Deprecated; same as UNSUPPORTED_ACCESS
+    EMBER_ZCL_STATUS_UNSUPPORTED_ENDPOINT     = 0x7F,
+    EMBER_ZCL_STATUS_MALFORMED_COMMAND        = 0x80,
+    EMBER_ZCL_STATUS_UNSUPPORTED_COMMAND      = 0x81,
+    EMBER_ZCL_STATUS_DEPRECATED82             = 0x82,
+    EMBER_ZCL_STATUS_DEPRECATED83             = 0x83,
+    EMBER_ZCL_STATUS_DEPRECATED84             = 0x84,
+    EMBER_ZCL_STATUS_INVALID_COMMAND          = 0x85,
+    EMBER_ZCL_STATUS_INVALID_FIELD            = 0x85, // Deprecated; same as INVALID_COMMAND
+    EMBER_ZCL_STATUS_UNSUPPORTED_ATTRIBUTE    = 0x86,
+    EMBER_ZCL_STATUS_CONSTRAINT_ERROR         = 0x87,
+    EMBER_ZCL_STATUS_INVALID_VALUE            = 0x87, // Deprecated; same as CONSTRAINT_ERROR
+    EMBER_ZCL_STATUS_UNSUPPORTED_WRITE        = 0x88,
+    EMBER_ZCL_STATUS_READ_ONLY                = 0x88, // Deprecated; same as UNSUPPORTED_WRITE
+    EMBER_ZCL_STATUS_RESOURCE_EXHAUSTED       = 0x89,
+    EMBER_ZCL_STATUS_INSUFFICIENT_SPACE       = 0x89, // Deprecated; same as RESOURCE_EXHAUSTED
+    EMBER_ZCL_STATUS_DUPLICATE_EXISTS         = 0x8A,
+    EMBER_ZCL_STATUS_NOT_FOUND                = 0x8B,
+    EMBER_ZCL_STATUS_UNREPORTABLE_ATTRIBUTE   = 0x8C,
+    EMBER_ZCL_STATUS_INVALID_DATA_TYPE        = 0x8D,
+    EMBER_ZCL_STATUS_DEPRECATED8E             = 0x8E,
+    EMBER_ZCL_STATUS_UNSUPPORTED_READ         = 0x8F,
+    EMBER_ZCL_STATUS_DEPRECATED90             = 0x90,
+    EMBER_ZCL_STATUS_DEPRECATED91             = 0x91,
+    EMBER_ZCL_STATUS_DATA_VERSION_MISMATCH    = 0x92,
+    EMBER_ZCL_STATUS_DEPRECATED93             = 0x93,
+    EMBER_ZCL_STATUS_TIMEOUT                  = 0x94,
+    EMBER_ZCL_STATUS_RESERVED95               = 0x95,
+    EMBER_ZCL_STATUS_RESERVED96               = 0x96,
+    EMBER_ZCL_STATUS_RESERVED97               = 0x97,
+    EMBER_ZCL_STATUS_RESERVED98               = 0x98,
+    EMBER_ZCL_STATUS_RESERVED99               = 0x99,
+    EMBER_ZCL_STATUS_RESERVED9A               = 0x9A,
```

Comment:
Please add `EMBER_ZCL_STATUS_RESERVED9B               = 0x9B,`

Reply:
0x9B is not actually explicitly reserved in the spec. That is, it's not a "has been used in the past in ZCL" value, and can be assigned to an error code in the future (unlike 0x99, 0x9A, etc).